cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/weights/pallet_bridge_relayers.rs

```diff
@@ -49,19 +51,29 @@ use core::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_bridge_relayers::WeightInfo for WeightInfo<T> {
 	/// Storage: `BridgeRelayers::RelayerRewards` (r:1 w:1)
-	/// Proof: `BridgeRelayers::RelayerRewards` (`max_values`: None, `max_size`: Some(102), added: 2577, mode: `MaxEncodedLen`)
+	/// Proof: `BridgeRelayers::RelayerRewards` (`max_values`: None, `max_size`: Some(74), added: 2549, mode: `MaxEncodedLen`)
 	/// Storage: `System::Account` (r:1 w:1)
 	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
 	fn claim_rewards() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `272`
+		//  Measured:  `245`
 		//  Estimated: `3593`
-		// Minimum execution time: 52_499_000 picoseconds.
-		Weight::from_parts(53_659_000, 0)
+		// Minimum execution time: 51_920_000 picoseconds.
+		Weight::from_parts(53_320_000, 0)
 			.saturating_add(Weight::from_parts(0, 3593))
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+	/// Storage: `Benchmark::Override` (r:0 w:0)
+	/// Proof: `Benchmark::Override` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	fn claim_rewards_to() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 18_446_744_073_709_551_000 picoseconds.
+		Weight::from_parts(18_446_744_073_709_551_000, 0)
```

Comment:
Is this okay?

Reply:
yes, Weight::MAX, because `claim_rewards_to` benchmark is not supported yet, because of:
https://github.com/paritytech/polkadot-sdk/pull/7492/files#diff-fe0ccabcf75160287fb96b1e0b334c0e71cf3a527a13e7d452e27814ca95f853R80-R82
```
BridgeRewardKind::Snowbridge =>
				Err(sp_runtime::DispatchError::Other("Not implemented yet, check also `fn prepare_rewards_account` to return `alternative_beneficiary`!")),
```